### PR TITLE
Feature/advanced indexing

### PR DESCRIPTION
**Pull request by garyluu**
A new test that:
- creates a tool
- refreshes the tool
- perform a full re-index
- add labels to the tool

a specified number of times.  Eventually, the elastic manager should be able to see which tools have changed and do partial re-index instead of doing a full one every time.

This will fail on Travis until the swagger fix is in.

### CODE REVIEW

**Comment by denis-yuen** [comment]
note re-run tests after #843 is squashed/rebased

**Comment by denis-yuen** [comment]
To be clear, this looks like this adds a test and a future foundation for advanced indexing. But actual implementation is in a future PR

**Comment by garyluu** [comment]
Yes, it only adds a test right now.

File: dockstore-integration-testing/src/test/java/io/dockstore/client/cli/AdvancedIndexingIT.java [hunk 1]
```diff
@@ -0,0 +1,274 @@
...
+
+import static junit.framework.TestCase.assertTrue;
+
+/**
+ * Focuses on creating a large amount of tools to test indexing
+ *
+ * @author aduncan
+ */
+public class AdvancedIndexingIT {
+    private static final int TOOL_COUNT = 10;
```
Comment (denis-yuen) [inline dockstore-integration-testing/src/test/java/io/dockstore/client/cli/AdvancedIndexingIT.java]: How long does this take to run on TravisCI anyway or is it skipped due to no ElasticSearch service?

**Comment by denis-yuen** [comment]
As discussed, this doesn't really need to be a test. This can just be a program that we run to benchmark our use of elastic search. (Add an entry in Dockstore primer or the readme.md)

**Pull request merged**